src/components/productForList/ProductForList.css

```
@@ -0,0 +1,84 @@
+.product-container {
+  display: flex;
+  height: 40px;
+  width: 100%;
+  border-radius: 8px;
+  margin-bottom: 10px;
+  background-color: #ffffff;
+
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.main-elements-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  width: 300px;
+}
+
+.color-mark {
+  width: 7px;
+  height: 100%;
+  border-top-left-radius: 8px;
+  border-bottom-left-radius: 8px;
+}
+
+.name-container {
+  width: 40%;
+}
+
+.item-name {
+  font-size: 12px;
+  font-weight: 600;
+  text-transform: capitalize;
+}
+
+.checked {
+  color: #68d918;
```

Bajo contraste entre el icono y el fondo del container.

```suggestion
  color: #c8c8c8;
```

---

igual relacionado con lo de arriba!